includes/admin/class-wc-stripe-inbox-notes.php

```diff
@@ -57,28 +57,26 @@ public static function notify_on_apple_pay_domain_verification( $verification_co
 			return;
 		}
 
+		try {
+			$data_store       = WC_Data_Store::load( 'admin-note' );
+			$failure_note_ids = $data_store->get_notes_with_name( self::FAILURE_NOTE_NAME );
+			// Delete all previously created, soft deleted and unactioned failure notes (Legacy).
+			while ( ! empty( $failure_note_ids ) ) {
+				$note_id = array_pop( $failure_note_ids );
+				$note    = WC_Admin_Notes::get_note( $note_id );
+				$note->delete();
+			}
+		} catch ( Exception $e ) {
+			return;
+		}
+
 		if ( $verification_complete ) {
 			if ( self::should_show_marketing_note() && ! wp_next_scheduled( self::POST_SETUP_SUCCESS_ACTION ) ) {
 				wp_schedule_single_event( time() + DAY_IN_SECONDS, self::POST_SETUP_SUCCESS_ACTION );
 			}
```

Comment:
I think the `return` can be removed.
I didn't know the reason for wrapping those in try/catch blocks, so I assumed if the `WC_Data_Store` or `WC_Admin_Notes` can fail, it will probably fail after the catch too in `should_show_marketing_note` or `create_failure_note`.

Reply:
Yeah, I'm not sure about that either. Let's assume for now that we might want it to run through that if statement; just like it did before 🙂